src/parser/scanner.lex

```diff
@@ -27,7 +27,6 @@ static constexpr size_t MAX_STRING = 4096;
 GO                          ([Gg][Oo])
 AS                          ([Aa][Ss])
 TO                          ([Tt][Oo])
-OR                          ([Oo][Rr])
```

Comment:
Why remove it?

Reply:
Previously `UPDATE OR INSERT`, but now changed to `UPSERT`.  So `OR` does not have to exist.

Reply:
Make sense